### PR TITLE
IMB-10: Set up skeleton ima repo

### PR DESCRIPTION
**Pull request by Rhodine-orleans-lindsay**
## What
- Add initial folders and files required to run the app locally
- Add initial folders and files required to run the app in branch/uat/stg/prod environments (some envars will need updated once they are provided)
- Add drone.yml
- Update readMe
## Why
- Initial repo set up for ima

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,527 @@
+---
+kind: pipeline
+name: default
+type: kubernetes
+
+environment:
+  APP_NAME: ima
+  PROD_ENV: ima
+  STG_ENV: ima-stg
+  UAT_ENV: sas-ima-uat
```
Comment (SarahJaneLuff) [inline .drone.yml:10]: should this be ima-uat?

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,527 @@
...
+  BRANCH_ENV: sas-ima-notprod
```
Comment (SarahJaneLuff) [inline .drone.yml:11]: ima-branch?

Comment (Rhodine-orleans-lindsay) [reply .drone.yml]: Looking at [IMB-8](https://collaboration.homeoffice.gov.uk/jira/browse/IMB-8), it is now sas-ima-branch so I have updated it and have updated ima-stg to sas-ima-stg

File: .drone.yml (hunk 1 above)
Comment (Rhodine-orleans-lindsay) [reply .drone.yml:10]: Looking at [IMB-8](https://collaboration.homeoffice.gov.uk/jira/browse/IMB-8), it is now sas-ima-uat

File: kube/file-vault/file-vault-deployment.yml [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+                secretKeyRef:
+                  name: keycloak-client
+                  key: id
+            - name: PROXY_LISTEN
+              value: 127.0.0.1:3001
+            - name: PROXY_UPSTREAM_URL
+              value: "http://127.0.0.1:3000"
+            - name: PROXY_REDIRECTION_URL
+            {{ if eq .KUBE_NAMESPACE .PROD_ENV }}
+              value: https://fv-rac.sas.homeoffice.gov.uk
```
Comment (SarahJaneLuff) [inline kube/file-vault/file-vault-deployment.yml:137]: Need to check these namespaces for file-vault, they will need to change for ima

File: kube/file-vault/file-vault-deployment.yml [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+                {{ else }}
+                name: configmap
+                {{ end }}
+          env:
+            # because the internal virus scanner has a self signed cert!
+            - name: NODE_TLS_REJECT_UNAUTHORIZED
+              value: "0"
+            - name: FILE_VAULT_URL
+            {{ if eq .KUBE_NAMESPACE .PROD_ENV }}
+              value: https://fv-rac.sas.homeoffice.gov.uk
```
Comment (SarahJaneLuff) [inline kube/file-vault/file-vault-deployment.yml:61]: Need to check these namespaces for file-vault, they will need to change for ima

**Review by SarahJaneLuff — COMMENTED** [review]
Looks good, just a few points.

**Pull request merged**